Dockerfile

```diff
@@ -154,6 +154,8 @@ ENTRYPOINT ["./ecs-resource-agent"]
 # Builds the vSphere K8s cluster resource agent image
 FROM public.ecr.aws/amazonlinux/amazonlinux:2 as vsphere-k8s-cluster-resource-agent
 
+ENV K8S_1_28_SUPPORT=true
+
```

Comment:
I think it would be best to let the testsys agent themselves determine the env var to populate its environment with.